src/mixins/itext_key_behavior.mixin.js

```diff
@@ -140,15 +137,9 @@ fabric.util.object.extend(fabric.IText.prototype, /** @lends fabric.IText.protot
 
   /**
    * Handles keyup event
-   * We handle KeyUp because ie11 and edge have difficulties copy/pasting
-   * if a copy/cut event fired, keyup is dismissed
    * @param {Event} e Event object
    */
   onKeyUp: function(e) {
-    if (!this.isEditing || this._copyDone || this.inCompositionMode) {
```

Comment:
the fact that there is inCompositionMode is alarming.
We need to verify why we were blocking paste/copy in composition mode.

Reply:
I am guessing here but it the copy was synthetic so when run with `inCompositionMode` it hit some edge case with `ctrl + v/c`

Reply:
Now browser is in charge of deciding what the action is so less edge cases for all. Hopefully

Reply:
Ohhh I am not certain anymore

Reply:
> We need to verify why we were blocking paste/copy in composition mode.

It is blocked for all mouse up when not in editing mode. Maybe because of the char suggestion toolbar?

Reply:
If so I should revert